cmd/podman/login.go

```diff
@@ -90,6 +91,19 @@ func loginCmd(c *cliconfig.LoginValues) error {
 	}
 
 	ctx := getContext()
+
+	var password string
+	var stdinPasswordStrBuilder strings.Builder
+	if c.Flag("password-stdin").Changed {
+		if c.Username == "" {
+			return errors.Errorf("Must provide --username with --password-stdin")
+		}
+		scanner := bufio.NewScanner(os.Stdin)
```

Comment:
Is there a more secure version of this? I'm thinking we probably want a version that replaces what's being typed with * so we don't display passwords as they're being typed in

Reply:
Doesn't this enable

echo MYPASSWORD | podman login --passwd-stdin

Reply:
@rhatdan Yes, it enables that